app/src/main/java/com/example/jwplayerdemo/jwsettings/JWViewModel.java

```diff
@@ -33,10 +37,24 @@
 
 public class JWViewModel extends ViewModel {
 
+    private final String DEFAULT_URL = "https://content.jwplatform.com/videos/8TbJTFy5-cIp6U8lV.mp4";
```

Comment:
This can be `static` since it doesn't differ between instances of `JWViewModel`, making it `static` will only allocate memory for it once across all instances so it's a little more resource efficient.

Reply:
^ ah very insightful, okay!